airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -270,34 +271,52 @@ def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[
         return params
 
 
-class Repositories(GithubStream):
+# Below are full refresh streams
+
+
+class Assignees(GithubStream):
     """
-    This stream is technical and not intended for the user, it is used only to obtain repositories for organizations.
-    API docs: https://docs.github.com/en/rest/reference/repos#list-organization-repositories
+    API docs: https://docs.github.com/en/rest/reference/issues#list-assignees
     """
 
-    def __init__(self, organizations: List[str], **kwargs):
-        super(GithubStream, self).__init__(**kwargs)
-        self.organizations = organizations
 
-    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
-        for organization in self.organizations:
-            yield {"organization": organization}
+class PullRequestStats(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request
+    """
 
-    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
-        return f"orgs/{stream_slice['organization']}/repos"
+    fields_to_minimize = ("merged_by",)
 
-    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
-        yield from response.json()
+    @property
+    def record_keys(self) -> List[str]:
+        return list(self.get_json_schema()['properties'].keys())
 
+    def path(
+        self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> str:
+        return f"repos/{stream_slice['repository']}/pulls/{stream_slice['pull_request_number']}"
 
-# Below are full refresh streams
+    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, Any]]]:
+        for stream_slice in super().stream_slices(**kwargs):
+            pull_requests_stream = PullRequests(authenticator=self.authenticator, repositories=[stream_slice["repository"]], start_date="")
```

Comment:
I based this off the Reviews stream. Would it be beneficial for both streams to pass the config's start_date to reduce the number of API calls these streams make?

Reply:
To reduce the number of requests for Pull Refresh Stream, we use caching (`vcr` library)